examples/flatris/webpack/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+import path from 'path';
+import webpack from 'webpack';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+
+// This config doesn't have entry and output set up because it's not meant to
+// work standalone. Cosmos adds an entry & output when extending this.
+module.exports = {
+  devtool: 'eval',
+  resolve: {
+    extensions: ['', '.js', '.jsx'],
```

Comment:
`root: path.resolve('./path_to_root')`

Reference: http://webpack.github.io/docs/configuration.html#resolve-modulesdirectories

This will remove the `../../../../../../constants/my_awesome_constant.js`